src/MediatR/Mediator.cs

```diff
@@ -40,6 +40,29 @@ public Task<TResponse> Send<TResponse>(IRequest<TResponse> request, Cancellation
             return handler.Handle(request, cancellationToken, _serviceFactory);
         }
 
+        public Task<object> Send(object request, CancellationToken cancellationToken = default)
+        {
+            if(request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+            var requestType = request.GetType();
+            var requestInterfaceType = Array.Find(requestType.GetInterfaces(),

```

Comment:
Why not `requestType.GetInterfaces().FirstOrDefault(...)`?

Reply:
Sorry for delay, replaced the call with suggestion.